docs/notebooks/us-names.ipynb

```diff
@@ -808,6 +808,7 @@
     "    plt.xlabel('Year')\n",
     "    plt.ylabel('Count')\n",
     "    plt.show()"
```

Comment:
I believe `\n` and comma are missed here, colab gives me an error, when I'm trying to open the NB
```suggestion
    "    plt.show()\n",
```

See also CI failure
https://github.com/snad-space/coniferest/actions/runs/19936717831/job/57165280369?pr=299